### PR TITLE
Update patternfly core to 5.2.0-prerelease.21

### PR DESCRIPTION
**Pull request by adamkankovsky**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
   },
   "dependencies": {
     "@patternfly/patternfly": "5.1.0",
```
Comment (KKoukiou) [inline package.json:46]: The @patternfly group needs to be bumped in non-minor version updates all together. 

In other words you need to bump all 4 depedencies.

**Review by KKoukiou — CHANGES_REQUESTED** [review]

**Review by KKoukiou — COMMENTED** [review]
And please remove webui prefix.

File: package.json (hunk 1 above)
Comment (adamkankovsky) [reply package.json:46]: I cant change a version of a react-log-viewer because most recent version is 5.1.0-prerelease.1

Comment (KKoukiou) [reply package.json]: you can try if the 'mix' of versions works or wait. Up to you.

**Review by KKoukiou — APPROVED** [review]

**Pull request merged**